lib/core/check_for_lang_updates.js

```diff
@@ -61,6 +67,15 @@
         logger.error(`did not find entry for language '${name}' in manifest`);
         errored = true;
       }
+    } else if (type === "langs") {
+      for (let i = 0; i < names.length; i++) {
+        if (!manifest.langs.includes(names[i])) {
+          logger.error(`did not find entry for language '${names[i]}' in manifest`);
```

Comment:
i would suggest making this into a fatal error.

Reply:
This would mean that if you want to add 5 languages, of which one is non-existant the entire command would fail and no languages are added. Resorting to a non-fatal error would mean the 4 other languages in this example would still be added

Reply:
feel free to log all invalid languages but ill still stand by this should be fatal. this being fatal would mimic the behavior of npm